charts/hono/values.yaml

```diff
@@ -30,6 +30,12 @@ honoContainerRegistry: "index.docker.io"
 # be added to every deployment and statefulset.
 imagePullSecrets: []
 
+probes:
```

Comment:
> I like the overall approach :-) However, in a service release, we cannot simply remove the existing properties for configuring the probe delays. Instead, we should keep them an mark them as deprecated, providing a hint what to use instead. We can, however, keep them empty and implement logic, that uses the old probes if the values are set explicitly, but otherwise uses your new probes based on the values of the new properties in values.yaml. WDYT?

yeah my bad, i'll make it backwards compatible